packages/app/src/components/navigation/internalPages/index.tsx

```diff
@@ -0,0 +1,60 @@
+import classnames from 'classnames';
+import classNames from 'classnames';
+import React from 'react';
+import { Props as BaseProps } from '~/components';
+import BarsOutLineIcon from '~/components/icon/bars/outline';
+import HomeOutlineIcon from '~/components/icon/home/outline';
+import Link from '~/components/link';
+import { INTERNAL_PAGE_PATHS } from '~/constants';
+import { useI18n, useTranslation } from '~/hooks/i18n';
+import { Pathname, URL as _URL } from '~/types';
+
+const paths: {
+  to: Pathname | _URL;
+  label: string;
+  icon: JSX.Element;
+}[] = [
+  {
+    to: INTERNAL_PAGE_PATHS.ROOT,
+    label: 'internalPagePaths.root',
+    icon: <HomeOutlineIcon className="w=em" />,
+  },
+  {
+    to: INTERNAL_PAGE_PATHS.ENDPOINTS,
+    label: 'internalPagePaths.endpoints',
+    icon: <BarsOutLineIcon className="w=em" />,
```

Comment:
Perhaps the classname w=em is a typo?

Reply:
[be7d2ce](https://github.com/cam-inc/viron/pull/674/commits/be7d2cebf1694963cd79458bb7a8028ff2f4feb8)